src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -37,18 +37,12 @@
 
 
     public BattleshipModel() {
-        computer_clipper = new Civilian("Computer_Clipper", 3, new Coordinate(5, 1), new Coordinate(5, 3));
-        computer_dinghy = new Civilian("Computer_Dinghy", 1, new Coordinate(10, 10), new Coordinate(10, 10));
-        computer_aircraftCarrier = new Military(false,"Computer_AircraftCarrier",5, new Coordinate(2,2),new Coordinate(2,6));
-        computer_battleship = new Military(true,"Computer_Battleship",4, new Coordinate(2,8),new Coordinate(5,8));
-        computer_submarine = new Military(true,"Computer_Submarine",2, new Coordinate(9,6),new Coordinate(9,7));
-
-        playerHits = new ArrayList<>();
-        playerMisses= new ArrayList<>();
-        computerHits = new ArrayList<>();
-        computerMisses= new ArrayList<>();
-
-        shipSquares = new ArrayList<>();
+        System.out.println("This is the parent BattleshipModel default constructor.");
```

Comment:
@aoleson I just wanted both classes' default contructors to serve the same purpose of placing AI ships. That way when the hard mode class's attributes are inherited, all the ships start at 0. But as we discussed, since the default mode is easy and hard mode shares the exact same ship instances across both classes, hard mode's ships get set with easy mode's hardcoded locations no matter what. So yeah.